scripts/system_tests_utils.ts

```diff
@@ -259,7 +259,7 @@ const walletsOnWorker = 20;
 if (tempAccountList.length === 0) {
 	tempAccountList = accountsString;
 }
-let currentIndex = 0;
+let currentIndex = Math.floor(Math.random() * tempAccountList.length);
```

Comment:
What's the purpose of this change?

Reply:
Tests for every package start with 0 and gets accounts from a list. So, to make accounts equal in use, I've added a random start index